DFC.TestAutomation.UI/TestSupport/Configurator.cs

```diff
@@ -6,55 +6,78 @@
 
 namespace DFC.TestAutomation.UI.TestSupport
 {
-    public static class Configurator
+    public  class Configurator
```

Comment:
Naming convention for properties is't clear. Do we want _variableName or VariableName?

Reply:
so far, I've just adopted what was already there. Happy to review as part of a wider look at the project